integration/kubernetes/nginx.bats

```diff
@@ -18,14 +18,18 @@
 load "${BATS_TEST_DIRNAME}/lib.sh"
 
 setup() {
-	nginx_image="nginx"
-	busybox_image="busybox"
-	service_name="${nginx_image}-service"
+	image_registry="docker.io"
```

Comment:
@chavafg  -- should we move this to the crio.conf the we install?  If folks want to be able to follow the script to install CRIO and expect it to work (with K8S), they'll want this set.

Reply:
@egernst fixed :)